app/models/list.rb

```diff
@@ -1,7 +1,9 @@
 class List < ApplicationRecord
+  acts_as_taggable
+
   has_and_belongs_to_many :homepages
   belongs_to :user
-
+ 
```

Comment:
removed, although i think the space already there.  It looked like this
`''  has_and_belongs_to_many :homepages
  belongs_to :user 
  
  has_many :papers, through: :references
  has_many :references`''